packages/fxa-auth-server/package.json

```diff
@@ -113,7 +113,7 @@
     "moment-timezone": "^0.5.35",
     "mozlog": "^3.0.2",
     "mysql": "^2.18.1",
-    "node-uap": "https://github.com/mozilla-fxa/node-uap.git#96dc1f9f224422ec184395b6408cd1fc40ee452a",
+    "node-uap": "https://github.com/dschom/node-uap.git#1e0a696e4ad351fdc7dad2b704af2102467cf958",
```

Comment:
This needs to be directed back at the mozilla-fxa/node-uap repo, but I need this change approved. https://github.com/mozilla-fxa/webcrypto-liner/pull/1.